<launch>
  <param name="robot_description" textfile="$(find pegasus_sim)/viz/qcopter.urdf"  />
  <group ns="pegasus_sim">
    <rosparam command="load" file="$(find pegasus_sim)/param/simulation_params.yaml"  />
    <node name="eom" pkg="pegasus_sim" type="equations_of_motion" output="screen"   />
  </group>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find pegasus_sim)/viz/qcopter.rviz" required="true"/>
</launch>
